google/resource_container_cluster.go

```diff
@@ -447,19 +472,35 @@ func resourceContainerClusterCreate(d *schema.ResourceData, meta interface{}) er
 	}
 
 	if v, ok := d.GetOk("additional_zones"); ok {
+		if isRegionalCluster {
+			return fmt.Errorf("'additional_zones' is incompatible with regional clusters")
```

Comment:
This should be doable with `ConflictsWith` in the schema

Reply:
Thank you. That should simplify this.